html/pfappserver/root/src/views/Status/dashboard/_config/portal.js

```diff
@@ -0,0 +1,51 @@
+import { modes, libraries } from '../_components/Chart'
+
+export default [
+  {
+    name: 'Portal', // i18n defer
+    groups: [
+      {
+        name: 'Captive Portal Responses', // i18n defer
+        items: ['1xx', '2xx', '3xx', '4xx', '5xx', 'other'].map(http_code => {
+          return {
+            title: `${http_code} responses`, // i18n defer
```

Comment:
This won't be translated. We should stick to `http_code + ' responses'`, or add the possibility to pass an object to `title` with a placeholder and variables.